FEATURE-LIFECYCLE.md

```diff
@@ -87,7 +87,7 @@ Enabled by default</td>
   </tr>
   <tr>
     <td>Completeness</td>
-    <td>Varies</td>
+    <td>Feature has limited capabilities, but acts as a proof-of-concept for additional ideas to follow.</td>
```

Comment:
don't use but in a governance doc please :)

`Feature has limited capabilities. This feature may act as a proof of concept.`